admin-dev/themes/new-theme/js/app/pages/translations/components/principal/index.vue

```diff
@@ -154,16 +157,39 @@
       },
     },
     methods: {
+      /**
+       * Dispatch the event to change the page index,
+       * get the translations and reset the modified translations into the state
+       * @param {Integer} pageIndex
```

Comment:
Just FYI, there's not `Integer` type in JS, in the future you should use `Number`.